ui/component/recommendedContent/view.jsx

```diff
@@ -68,14 +72,16 @@ export default class RecommendedContent extends React.PureComponent<Props> {
         isBodyList
         title={__('Related')}
         body={
-          <ClaimList
-            isCardBody
-            type="small"
-            loading={isSearching}
-            uris={recommendedContent}
-            injectedItem={!isAuthenticated && IS_WEB && <Ads type="video" small />}
-            empty={__('No related content found')}
-          />
+          <WaitUntilOnPage lastUpdateDate={this.lastReset}>
```

Comment:
Should this prop be added to the `WaitUntilOnPage` wrapper around `CommentList`?

Reply:
Ah, missed it.  For some reason, I kept thinking that one is the Channel Comments.